src/editor/Collaboration.js

```diff
@@ -7,11 +7,16 @@ define(function (require, exports, module) {
     var Path            = require("filesystem/impls/filer/FilerUtils").Path;
     var EditorManager   = require("editor/EditorManager");
     var CommandManager  = require("command/CommandManager");
+    var DocumentManager = require("document/DocumentManager");
+    var FileUtils       = require("file/FileUtils");
```

Comment:
you don't really need this. You can use the same function from `FilerUtils` instead